deploy/charts/cert-manager/templates/webhook-psp.yaml

```
@@ -27,7 +27,10 @@ spec:
   - 'projected'
   - 'secret'
   - 'downwardAPI'
-  hostNetwork: false
+  hostNetwork: {{ .Values.webhook.hostNetwork }}
+  hostPorts:
+  - max: {{ .Values.webhook.securePort }}
+    min: {{ .Values.webhook.securePort }}
```

Maybe it would be better this:

```suggestion
  hostNetwork: {{ .Values.webhook.hostNetwork }}
  {{ if .Values.webhook.hostNetwork }}
  hostPorts:
  - max: {{ .Values.webhook.securePort }}
    min: {{ .Values.webhook.securePort }}
  {{ end }}
```

This way it won't add `hostPort` to the psp unless it really needs it. @Kirill-Garbar @HardBrainer

---

I totally agree. Thank you.